frontend/src/components/UserPopUp/index.tsx

```diff
@@ -1,43 +1,51 @@
 import styles from './styles.module.sass';
 import React, { FunctionComponent } from 'react';
-import { IUser } from '../../common/models/user/IUser';
 import { blockPosition } from '../../common/types/types';
-import Image from 'react-bootstrap/Image';
 import Popover from 'react-bootstrap/Popover';
 import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
-import Button from 'react-bootstrap/Button';
-import { userLogoDefaultUrl } from 'common/configs/defaults';
+import { getRefreshToken } from '../../common/helpers/storageHelper';
+import { removeToken } from '../../services/authService';
+import { Routes } from 'common/enums/Routes';
 
 interface IProps {
-  user: IUser;
   trigger: () => React.ReactElement;
   id: string;
   placement: blockPosition;
   onEditProfileClick: () => void;
 }
 
-const UserPopUp: FunctionComponent<IProps> = ({ user, trigger, id, placement, onEditProfileClick }) => {
-  const popOver = (
-    <Popover id={id}>
-      <Image
-        className={styles.userAvatarBig}
-        src={user.imageUrl || userLogoDefaultUrl}
-        alt="User avatar big"
-        thumbnail
-      />
-
-      <Popover.Content>
-        <p className={styles.usernameHeader}>{user.fullName}</p>
-        <p>Some info</p>
-        <p>Some othe info</p>
-        <Button variant="primary" onClick={onEditProfileClick}>Edit profile</Button>
-      </Popover.Content>
+const UserPopUp: FunctionComponent<IProps> = ({ trigger, id, placement, onEditProfileClick }) => {
+  const onSingOut = () => {
+    removeToken(getRefreshToken());
+    localStorage.clear();
+    window.location.href = Routes.SignIn;
+  };
 
+  const popOver = (
+    <Popover id={id} className={styles.panepPopUp}>
+      <a href="#0" className={styles.panelSelect}>
+        <span className={styles.panelText}>Preferenses</span>
+      </a>
+      <a href="#0" className={styles.panelSelect} onClick={onEditProfileClick}>
+        <span className={styles.panelText}>View Profile</span>
+      </a>
+      <a href="#0" className={styles.panelSelect}>
+        <span className={styles.panelText}>Invite People</span>
+      </a>
+      <a href="#0" className={styles.panelSelect}>
+        <span className={styles.panelText}>Add workspace</span>
+      </a>
+      <a href="#0" className={styles.panelSelect}>
+        <span className={styles.panelText}>Switch workspace</span>
+      </a>
+      <a href="#0" className={styles.panelSelect} onClick={onSingOut}>
+        <span className={styles.panelText}>Sign Out</span>
+      </a>
```

Comment:
use React <Link> component instead of <"a"> tag. Don't use links without anchors (or with mocks like "#0"), instead add styles to usual text